### ISSUE TITLE
`viewElement.scroll` slows down on repeated calls when reusing the Detox element

### ISSUE THREAD

**Issue by yurijmikhalevich** [wix/Detox#2034]
### Description
`viewElement.scroll` slows down on repeated calls when reusing the Detox element.

- [x] I have tested this issue on the latest Detox release and it still reproduces

#### Reproduction

Code with a slow down:
```
  const scrollView = element(by.id('scrollViewId'));
  while (true) {
    await scrollView.scroll(400, 'down');
    console.log('scrolled');
  }
  // it slow downs for me after 10 or so scrolls
```

It slows down by introducing a noticeable (more than a few seconds) lag before doing the scroll.

This will work well:
```
  while (true) {
    await element(by.id('scrollViewId')).scroll(400, 'down');
    console.log('scrolled');
  }
  // no slow downs
```

The first snippet works well on Android. Please, either fix that or document.

#### Expected behavior
No scroll slowdowns.

#### Environment (please complete the following information):
 - Detox: 16.4.0
 - React Native: 0.61.5
 - Node: 14.1.0
 - Device: MacBook with Catalina 10.15.4
 - Xcode: 11.4.1
 - iOS: 13.4 (emulator)
 - macOS: 10.15.4

#### If you are experiencing a timeout in your test
- [x] I have followed the instructions under [Lower-level Idling Resources Debug](https://github.com/wix/Detox/blob/master/docs/Troubleshooting.Synchronization.md#lower-level-idling-resources-debug-ios-only) and am providing the log below: [ios.log](https://github.com/wix/Detox/files/4611363/ios.log)

#### If you are seeing a build problem (e.g. during `npm install`)
- [ ] I am providing the `npm install` log below:

#### Device and verbose Detox logs
- [x] I have run my tests using the `--loglevel trace` argument and am providing the verbose log below: [ios-trace.log](https://github.com/wix/Detox/files/4613527/ios-trace.log)

**Comment by LeoNatan** [comment]
@rotemmiz Could this be some how "accumulating" more and more stuff as time goes by? Would this be fixed in invoke2?

@yurijmikhalevich Does this reproduce in Android as well?

**Comment by yurijmikhalevich** [comment]
@LeoNatan no, it seems to be fine on Android.

**Comment by LeoNatan** [comment]
@yurijmikhalevich Could you please post a trace log, as requested in the template, while reproducing they issue?

**Comment by yurijmikhalevich** [comment]
@LeoNatan added log of the laggy version to the OP.

**Comment by LeoNatan** [comment]
Thanks but that’s not the log I want. 

Please provide the `--loglevel trace` log.

**Comment by yurijmikhalevich** [comment]
@LeoNatan uploaded the trace log.

**Comment by LeoNatan** [comment]
We’re working on a new implementation of actions, which should be merged relatively soon. Let’s see if this reproduces there.

**Comment by yurijmikhalevich** [comment]
@LeoNatan great! Thank you! Please, let me know when it will be out so I'll test it.

I also noticed that `tap` action is much slower on iOS compared to Android. I wonder if it's due to the platform limitations or if there is a way to improve that.

**Comment by LeoNatan** [comment]
Let's see how it performs in the new system.

**Comment by yurijmikhalevich** [comment]
@LeoNatan is the new system ready? Should I test it again?

**Comment by LeoNatan** [comment]
The new system has not been deployed yet. You can follow the status here: https://github.com/wix/Detox/pull/2098

**Comment by LeoNatan** [comment]
Please test on latest Detox version and report back. Thanks

**Comment by yurijmikhalevich** [comment]
@LeoNatan the issue is fixed in 17.0.2.

**Comment by LeoNatan** [comment]
Great, thanks!
